server/mails/fr/mail_invitation.jade

```diff
@@ -36,7 +36,7 @@ html(style="height:100%;")
             tr(style="padding-top: 10px; text-align: center; font-size: 0.85em; font-style: italic; color: #999;")
               td
                 p
-                  | Envoyé depuis le&nbsp;
+                  | Envoyé depuis le 
```

Comment:
I think as long as the repo have an [editorconfig file](https://github.com/cozy/cozy-calendar/blob/master/.editorconfig#L15) that explicitely says that jade files must not be trimmed, It's OK to not use useless `&nbsp` that are unbreakables but rather use true whitespaces at end of line. The tools should not drive our code (and there is many editorconfig plugins for many IDEs :))